src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -202,11 +202,6 @@ internal override void ValidateOptions(ArrayBuilder<Diagnostic> builder)
 
         internal bool IsFeatureEnabled(MessageID feature)
         {
-            string featureFlag = feature.RequiredFeature();
```

Comment:
Please do not remove this from code from the compiler. It is intended to facilitate experimental code residing in branches of the compiler, even though the product code does not contain experiments.